### ISSUE TITLE
Wildcard in name doesn't auto enable on web

### ISSUE THREAD

**Issue by GioLogist** [visionmedia/debug#636]
More in #634

**Comment by GioLogist** [comment]
Thought it was a nice tough myself, IE: being able to use for logs. But yep, the nightmare of a dependency using it rather liberally would be a pain. Although, couldn't the same thing be said about logs in general?

**Comment by Qix-** [comment]
Yes and no. It depends on how they're handled. SLF4J handles this quite nicely - all logging facilities are handled by the end user.

Debug is similar, and it's assumed the the consumer (the user) controls the debugging entirely.

This is also why it's called `debug` and not `log` :wink:

**Comment by GioLogist** [comment]
> This is also why it's called debug and not log 😉

Hah, fair.
